### DISCUSSION TITLE
Manual splits

### DISCUSSION THREAD

**Discussion by SRobiscoUOC** [MIC-DKFZ/nnUNet#1426] [Q&A]
Hi,
First of all, I must congrat you for this amazing work.
I've a MRI dataset, that I've divided in train and test datasets. I'm not able to make nnU-Net work with my datasets, it makes it's own divisions. If I modify splits_final.json, nnU-Net throws an error telling that it doesn't find the files.

Please, do you have a tutorial of how to force nnU-Net V2 to  work as nnU-Net V1, using the datasets that we specify in dataset.json?

Thanks and regards,
Sara R.

**Comment by FabianIsensee** [comment]
Hey, nnU-Net v1 and v2 work the same way. dataset.json content was just ignored by nnU-Net v1.
The strategy is the following:

- place all your TRAINING images/labels in imagesTr/labelsTr
- place all your TEST images anywhere you like. You CAN use imagesTs/labelsTs for that in order to have everything in one place. nnU-Net will NOT use these folders, ever.

If you have specific training/validation splits you would like to use, create a splits_final.json file and place them there. See this documentation: https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/manual_data_splits.md

This file was a .pkl file in v1 and is now a .json file!
